ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -504,6 +504,16 @@ impl<'a> X64CodeGen<'a> {
                 }
                 self.store_new_local_float(iidx, WF0);
             }
+            BinOp::FAdd => {
+                self.load_operand_float(WF0, &lhs);
+                self.load_operand_float(WF1, &rhs);
+                match lhs.byte_size(self.m) {
+                    8 => dynasm!(self.asm; addsd Rx(WF0.code()), Rx(WF1.code())),
+                    4 => dynasm!(self.asm; addss Rx(WF0.code()), Rx(WF1.code())),
```

Comment:
Is it just me or these two arms (for `8` and `4`) identical?

Reply:
`addsd` vs `addss` :)

Reply:
Ah!